src/app/admin/charts/page.tsx

```diff
@@ -0,0 +1,194 @@
+'use client';
+
+import BarChart from '@/components/charts/BarChart';
+import useSession from '@/hooks/useSession';
+import { AdminChartStats } from '@/types/AdminChartStats';
+import { useState, useEffect } from 'react';
+import { BASE_URL } from 'src/services/api';
+
+const Charts: React.FC = () => {
+  const session = useSession();
+
+  // logic
+  const filters = [
+    {
+      name: 'Exames',
+      value: 'examsCharts'
+    },
+    {
+      name: 'Users Criados',
+      value: 'usersCreated'
+    },
+    {
+      name: 'Comments',
+      value: 'comments'
+    },
+    {
+      name: 'Reports',
+      value: 'reports'
+    }
+  ];
+
+  const subjects = [
```

Comment:
If a new subject is added to the db we will have to create a new release of the frontend to show it here. Is it a good idea?